CHANGELOG.rst

```diff
@@ -15,48 +15,146 @@ The following notes provide some details on what has been revised for each
 version in reverse chronological order (most recent version at the top
 of the list).
 
-3.2.2 (unreleased)
+3.3.0 (28-Sep-2021)
 ==================
+This version includes all the functionality needed to generate
+source catalogs, both point source and extended (segment) source
+catalogs, during single-visit mosaic (SVM) processing.  In fact,
+
+- Updated code to work with Python >= 3.7
+- GAIAeDR3 catalog now the initial catalog of choice for a posteriori alignment
+  during standard pipeline processing, as well as for SVM/MVM processing.
+- SVM/MVM processing will loop over catalogs, fit methods and fit geometries in
+  looking for successful fit, using first successful fit it computes.
+  - CATALOGS used: **GAIAeDR3**, **GSC242**, **2MASS** (in this order)
+  - methods: relative, image-by-image
+  - geometries: **rscale**, **rshift**, **shift** (each with different minimum cross-matches)
+- SVM processing will always generate both point source and extended source catalogs now
+  - point source catalog will be generated using TinyTim PSF-based detection
+  - extended source (segment) catalog will only have sources larger
+    than the PSF kernel deblended.
+  - catalog columns will closely resemble the Hubble Legacy Archive (HLA) catalogs columns
+- Grism and Prism exposures get same WCS as direct images
+- Grism/Prism exposures do not get aligned, but do get WCS correction from direct images
+- Added logic to handle visits where there are only Grism/Prism exposures, and nothing else
```

Comment:
new: _exposures, and no direct images_

This change just makes the description a bit more descriptive.

Reply:
Corrected in c11168d